### PR TITLE
[bc] Simplify change messages

### PR DESCRIPTION
**Pull request by catalinaperalta**
Some initial work to simplify the messages we use to report changes. We can continue to simplify them where needed.

Fixes https://github.com/Azure/azure-sdk-for-python/issues/36763

### CODE REVIEW

**Comment by catalinaperalta** [comment]
NOTE: if the changes look good as is, then I'll update the tests to use the updated message wording

File: scripts/breaking_changes_checker/breaking_changes_tracker.py [hunk 1]
```diff
@@ -42,63 +42,57 @@ class BreakingChangeType(str, Enum):
...
 class BreakingChangesTracker:
     REMOVED_OR_RENAMED_CLIENT_MSG = \
-        "The client '{}' was deleted or renamed in the current version"
+        "Deleted or renamed client '{}'"
     REMOVED_OR_RENAMED_CLIENT_METHOD_MSG = \
-        "The '{}' client method '{}' was deleted or renamed in the current version"
+        "'{}' client deleted or renamed method '{}'"
     REMOVED_OR_RENAMED_CLASS_MSG = \
-        "The model or publicly exposed class '{}' was deleted or renamed in the current version"
+        "Deleted or renamed '{}' model"
```
Comment (msyyc) [inline scripts/breaking_changes_checker/breaking_changes_tracker.py:49]: We shall keep consistent for the position of `client/method/model` before or after real name.
For line45 is `"Deleted or renamed client '{}'"`, and line49 is `"Deleted or renamed '{}' model"` which is inconsistent. 
I prefer put `client/...` before the real name.

Comment (msyyc) [inline scripts/breaking_changes_checker/breaking_changes_tracker.py:49]:
```suggestion
        "Deleted or renamed model '{}'"
```
We shall keep consistent for the position of `client/method/model` before or after real name.

Comment (msyyc) [reply scripts/breaking_changes_checker/breaking_changes_tracker.py:49]: And same for other MSG

File: scripts/breaking_changes_checker/breaking_changes_tracker.py [hunk 2]
```diff
@@ -42,63 +42,57 @@ class BreakingChangeType(str, Enum):
...
     REMOVED_OR_RENAMED_CLASS_METHOD_MSG = \
-        "The '{}' method '{}' was deleted or renamed in the current version"
+        "Deleted or renamed '{}.{}' method"
     REMOVED_OR_RENAMED_MODULE_LEVEL_FUNCTION_MSG = \
-        "The publicly exposed function '{}' was deleted or renamed in the current version"
+        "Deleted or renamed publicly exposed function '{}'"
     REMOVED_OR_RENAMED_POSITIONAL_PARAM_OF_METHOD_MSG = \
-        "The '{}' method '{}' had its parameter '{}' of kind '{}' deleted or renamed in the current version"
+        "'{}.{}' had its method parameter '{}' of kind '{}' deleted or renamed"
```
Comment (msyyc) [inline scripts/breaking_changes_checker/breaking_changes_tracker.py:55]:
```suggestion
        "'{}.{}' deleted or renamed its method parameter '{}' of kind '{}' "
```

Comment (msyyc) [reply scripts/breaking_changes_checker/breaking_changes_tracker.py:55]: same for other MSG with pattern `... had ... inserted/renamed/delelted`

File: scripts/breaking_changes_checker/breaking_changes_tracker.py (hunk 1 above)
Comment (catalinaperalta) [reply scripts/breaking_changes_checker/breaking_changes_tracker.py]: Made the message more uniform where possible.

**Review by msyyc — APPROVED** [review]
LGTM

**Pull request merged**